api_app/script_analyzers/observable_analyzers/checkdmarc.py

```diff
@@ -0,0 +1,37 @@
+import subprocess
+import logging
+import json
+
+from api_app.script_analyzers import classes
+
+logger = logging.getLogger(__name__)
+
+
+class CheckDMARC(classes.ObservableAnalyzer):
+    check_command: str = "checkdmarc"
+
+    def run(self):
+        try:
+            process = subprocess.Popen(
+                [self.check_command, self.observable_name],
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+            )
+            process.wait()
+            stdout, stderr = process.communicate()
+
+            dmarc_info = stdout.decode("utf-8"), stderr
+
+            dmarc_str = dmarc_info[0]
+
+            dmarc_json = json.loads(dmarc_str)
+
+            return dmarc_json
+
+        except OSError as e:
```

Comment:
The try-catch block here is redundant. The `run` function is itself wrapped into a try-catch block inside the `ObservableAnalyzer` class.